### PR TITLE
Add group amount to MGEN konnector

### PR DESCRIPTION
**Pull request by ptbrowne**
* fix amount that was not the right quantity
* add group amount
* remove old mgen files

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -14,10 +14,13 @@ const {
   log,
   saveFiles,
   saveBills,
-  errors
+  errors,
+  cozyClient
 } = require("cozy-konnector-libs");
 const moment = require("moment");
 const bluebird = require("bluebird");
+const rq = require('request')
```
Comment (doubleface) [inline src/index.js]: `rq` is not used anywhere

File: src/removeOldFiles.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+const lsDir = async folder => {
+  const limit = 100000
+  const folderId = folder._id
+  const data = await cozyClient.fetchJSON('GET', `/files/${folderId}?page[limit]=${limit}`, null, {
+    processJSONAPI: false
+  })
+  return data.included
+}
+
+const removeOldFiles = async fields => {
```
Comment (doubleface) [inline src/removeOldFiles.js]: `removeOldFiles` may remove files which are too old and will not be fetched in the new format. We should check the list of fetched bills and not remove files older than the oldest fetched bill

File: src/removeOldFiles.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  const folder = await cozyClient.files.statByPath(fields.folderPath)
+  const files = await lsDir(folder)
+  const oldMgenFiles = files.filter(isOldFile)
+  log('info', `Found ${oldMgenFiles.length} old MGEN files`)
+  return bluebird.map(oldMgenFiles, file => {
+    log('info', `Deleting file ${file.attributes.name}`)
+    return cozyClient.files.destroyById(file.id)
```
Comment (doubleface) [inline src/removeOldFiles.js]: What about bank operations which have references to these files ?

File: package.json [hunk 4]
```diff
@@ -17,13 +17,16 @@
...
   },
   "eslintIgnore": [
     "build"
   ],
+  "prettier": {
+    "semi": false,
+    "singleQuote": true
+  },
   "scripts": {
     "start": "node src/index.js",
```
Comment (doubleface) [inline package.json]: The `yarn test` command is missing

Comment (ptbrowne) [reply package.json]: good catch, thx !

**Pull request merged**